src/gui/sharelinkwidget.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>400</width>
-    <height>238</height>
+    <height>254</height>
```

Comment:
Is this necessary?

Reply:
yes, this require as we have added horizntal layout for dropdown which contains permission setting , please check the screenshot  
![image](https://user-images.githubusercontent.com/110465779/187384053-275fbbc0-f319-4465-8b14-af44906d3a0f.png)

Reply:
because of additional layout height of the widget has increased

Reply:
Please refer to my prior comment on having two separate menus